src/pages/baeyoonah/Login/Login.js

```diff
@@ -1,13 +1,77 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class LoginAh extends React.Component {
+  goToMain = () => {
+    this.props.history.push('/MainAh');
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      idInput: '',
+      pwInput: '',
+      defaultColor: { backgroundColor: 'skyblue' },
+      disabled: 'disabled',
+    };
+  }
+
+  handleInput = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
```

Comment:
- 구조 분해 할당 활용해 주시면 더 좋을 것 같습니다!
```suggestion
    const { name, value } = event.target;
    this.setState({
      [name]: value,
    });
```

Reply:
수정했습니다!